### PR TITLE
Routing and eventList

### PR DESCRIPTION
**Pull request by JoakimSM**

### CODE REVIEW

File: config/rewire.config.js [hunk 1]
```diff
@@ -135,6 +136,12 @@ function rewireAliases(config) {
     return config;
 }
 
+function addCssModuleRules(config) {
```
Comment (varl) [inline config/rewire.config.js]: Seems like a useless function?

File: public/i18n/module/i18n_module_en.properties [hunk 2]
```diff
@@ -22,3 +22,12 @@ abort=abort
```
Comment (varl) [inline public/i18n/module/i18n_module_en.properties]: Take a look at the [d2-i18n library](https://github.com/dhis2/d2-i18n#table-of-contents) we've standardised on for DHIS2 apps.

File: src/components/App/App.component.js [hunk 3]
```diff
@@ -1,9 +1,14 @@
 // @flow
 /* eslint-disable */
-import React, {Component} from 'react';
+import './app.css';
+import * as React from 'react';
```
Comment (varl) [inline src/components/App/App.component.js]: Isn't this the same as `imoprt React from 'react'`?

File: public/i18n/module/i18n_module_en.properties (hunk 2 above)
Comment (JoakimSM) [reply public/i18n/module/i18n_module_en.properties]: Yes, I'll look into it during the next sprint.

File: src/components/App/App.component.js (hunk 3 above)
Comment (JoakimSM) [reply src/components/App/App.component.js]: flow types are also imported when using * as React (among other stuff I guess)

File: config/rewire.config.js (hunk 1 above)
Comment (JoakimSM) [reply config/rewire.config.js]: I probably dozed off. Fixed.

**Comment by JoakimSM** [comment]
For now, you need to supply the orgUnit and program in the uri. something like: http://localhost:3000/#/programId=eBAyeGv0exc&orgUnitId=Rp268JB6Ne4

Notes:
- Icons are the wrong color due to the @material-ui/icons package importing 'material-ui/SvgIcon' which is the old material-ui package. **TODO: remove old material-ui package.** (node-modules/@material-ui/icons/utils/createSvgIcon.js line 15, change it to 'material-ui-next' and you'll see the correct colors)
- The only ordering currently working correctly is eventDate ('report date') because the api isn't returning the right results (server task created for this)

**Comment by Bekkalizer** [comment]
Is this pull request ready to merge?

**Pull request merged**